### ISSUE TITLE
Error on Create Data Import Job

### ISSUE THREAD

**Issue by GuillePardo** [aws-samples/amazon-forecast-samples#77]
At running the code for create data import I got this error:

An error occurred (InvalidInputException) when calling the CreateDatasetImportJob operation: No objects found in s3 path s3://bucket-name/AKIAVG6HB65434355, please ensure the path is specified correctly.

Can anyone help me with this? Thank you.

**Comment by chrisking** [comment]
Looks like your bucket name was not altered, please update it to the bucket you used.

**Comment by GuillePardo** [comment]
Hello Chrisking thankyou for your answe. I already checked the bucket name and evrything is correct. I don't know what else could be happening..

**Comment by chrisking** [comment]
So the error you provided earlier was a generic name, thus the question. Can you confirm you are using an IAM role that has permissions to access that bucket? Also what notebook are you using?

**Comment by GuillePardo** [comment]
That generic name was fulfilled with the real name of bucket at the notebook.

I am using a role with this access for Iam:

AmazonForecastFullAccess
AmazonS3FullAccess


![aws](https://user-images.githubusercontent.com/35585714/81096008-68719180-8ecb-11ea-87a2-c28e3f3a38d9.jpg)

**Comment by GuillePardo** [comment]
Iam using Jupyter Notebook

**Comment by chrisking** [comment]
Which notebook in this repository are you using?

**Comment by GuillePardo** [comment]
1.Getting_Data_Ready.ipynb

**Comment by GuillePardo** [comment]
line:

datasetImportJobName = 'EP_DSIMPORT_JOB_TARGET'
ds_import_job_response=forecast.create_dataset_import_job(DatasetImportJobName=datasetImportJobName,
                                                          DatasetArn=datasetArn,
                                                          DataSource= {
                                                              "S3Config" : {
                                                                 "Path": s3DataPath,
                                                                 "RoleArn": role_arn
                                                              } 
                                                          },
                                                          TimestampFormat=TIMESTAMP_FORMAT
                                                         )

**Comment by chrisking** [comment]
Interesting, validating that the notebooks from scratch work in a clean environment, will report back later.

**Comment by GuillePardo** [comment]
Thank you

**Comment by luseloso** [comment]
@GuillePardo can you confirm when you run a new cell printing the 's3DataPath' you are getting the correct bucket + key location?
The error can be when setting up this widget
text_widget_bucket = util.create_text_widget( "bucket_name", "input your S3 bucket name" )

A quick fix is to manually set the 'bucket_name' to your bucket directly without that text_widget_bucket

**Comment by GuillePardo** [comment]
That's right, I confirmed three times the name of the bucket and the key. Also I confirm the structure

s3://bucket-name/AKIAVG6HB65434355

**Comment by luseloso** [comment]
That key should be the name of the file you are dropping into S3, in this case should look something like

s3://bucket-name/elec_data/item-demand-time-train.csv

**Comment by GuillePardo** [comment]
Yes!! it Works.

Thank you very much!
